scania-digital-design-system/tegel
# feature: add motion to popover component

## **Describe pull-request**  
This PR applies animation "fade" to the popover-core component. This will be reflected in the popover-canvas, popover-menu.
popover-tooltip already have an animation from before, it should work as as it use to work.

## **Issue Linking:**  
_Choose one of the following options_
- **Jira:** : [CDEP-3703](https://tegel.atlassian.net/browse/CDEP-3703)    

## **How to test**  
_Provide detailed steps for testing, including any necessary setup._
1. Navigate to the AWS link
2. Go to popover-menu, popover-canvas & popover-tooltip
3. Make sure that the controls for animation works.

## **Checklist before submission**

## **Suggested test steps**

## **Screenshots**  
https://github.com/user-attachments/assets/42e3c66e-3e0d-4ec7-be9b-334ce7b41e1e

https://github.com/user-attachments/assets/e76d629b-dd24-4f8f-a9ef-718c826d472d


[CDEP-3703]: https://tegel.atlassian.net/browse/CDEP-3703?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
